react-native-app/views/SignUpView.tsx

```diff
@@ -6,17 +6,24 @@ import User from '../shared/models/User';
 import ServerFacade from '../backend/ServerFacade';
 import Colors from '../assets/constants/colors';
 import T from '../assets/constants/text';
+import { isEmpty } from '@aws-amplify/core';
```

Comment:
This looks like a great utility method. I'm just cautious about it coming from Amplify since we've started talking about moving away from Amplify.

When we do, it should be fairly simple to replace this with our own utility

Reply:
changed this to use isEmpty from lodash instead